File: Week1/cash.c
#include <cs50.h>
#include <stdio.h>

int main (void)
{
    int get_amount;
    do
    {
        get_amount=get_int("give cash : ");
    }
    while (get_amount < 0);

    int quarter=25;
    int dime=10;
    int nikkel=5;
    int penny=1;
    int count=0;
    while(get_amount != 0)
    {
        if(get_amount >= quarter)
        {
            get_amount=get_amount-quarter;
            count=count+1;
        }
        else if (get_amount >=dime && get_amount < quarter)
        {
            get_amount=get_amount - dime ;
            count = count+1 ;
        }
        else if (get_amount >=nikkel && get_amount < quarter && get_amount < dime)
        {
            get_amount=get_amount - nikkel ;
            count = count+1 ;
        }
        else if (get_amount >=penny && get_amount < nikkel && get_amount < quarter && get_amount < dime)
        {
            get_amount=get_amount - penny ;
            count = count+1 ;
        }
        else{
            break;
        }

    }

    printf("%i \n",count);
}

File: Python's Week/cash.py
# THese are prices
quarter=25
dime=10
nikkel=5
penny=1
count=0

# this is a condition to get Valid cash.
while(True):
    cash=int(input('get cash: '))
    if (0 > cash ):
        continue
    else:
        break

# THis is logic body
while (cash != 0):
    if (cash >= quarter):
        cash=cash-quarter
        count +=1

    elif (quarter > cash >= dime ):
        cash  = cash- dime
        count +=1

    elif (cash >= nikkel and cash < quarter and cash < dime):
        cash = cash - nikkel
        count +=1

    elif ( cash >= penny and cash < quarter and cash < dime and cash < nikkel):
        cash = cash - penny
        count +=1
    else:
        break

print(count)
